Makefile

```diff
@@ -0,0 +1,47 @@
+.PHONY: build run vendor
+SHELL := /bin/bash
+PKG := github.com/Clever/cron-admin
+SUBPKGS := $(addprefix $(PKG)/, db server)
+PKGS := $(PKG) $(SUBPKGS)
+EXECUTABLE := cron-admin
+GOLINT := $(GOPATH)/bin/golint
+GODEP := $(GOPATH)/bin/godep
+
+MONGO_TEST_DB ?= 127.0.0.1:27017
```

Comment:
change to `export MONGO_TEST_DB ?= 127.0.0.1:27017` and you don't need to have `MONGO_TEST_DB=$(MONGO_TEST_DB) ` below